server/bad_actor.py

```diff
@@ -138,8 +141,17 @@ def _slackify_all(self) -> list:
                         "value": json.dumps(self.transaction_data),
                     },
                 ],
-            },
-        ]
+            })
+        else:
+            slack_block.append({
+                "type": "section",
+                "fields": [{
+                    "type": "mrkdwn",
+                    "text": "Donation auto-rejected by CageAI"
```

Comment:
**note:** lol at CageAI. To your note in smells, that could make it sound a little more magical than it really is. I think leaving it as "Donation auto-rejected" would be just fine, although I do think this is a fun easter egg so also fine to leave it. I'll leave it to you! 🐣

Reply:
Yeah, that was more of a tongue-in-cheek commentary on the AI craze in general, but I see your point that that's not going to be the common understanding. Haha.